client/src/app/notifications/Notification.js

```diff
@@ -69,6 +70,29 @@ export default class Notification extends PureComponent {
     this.props.close();
   }
 
+  getContent() {
+    const {
+      content,
+      close
+    } = this.props;
+
+    // allow button to close notification
+    if (content && content.type === 'button') {
+      const contentWithButton = assign({},content);
```

Comment:
Can we add some test coverage for this in `NotificationSpec`?

Reply:
Added test case for this